### PR TITLE
Add --quiet parameter to docker-compose pull, using existing silent parameter

### LINKED ISSUE #3770: Option to suppress progress
There should be an option that still shows the build steps, but not the actual progress bars which spray a huge amount of output on CI servers since the docker commands expect that your using a terminal that has the control codes to write over previous lines.

### PR DESCRIPTION
**Pull request by jbarciauskas**
Resolves #3770

### CODE REVIEW

File: compose/cli/main.py [hunk 1]
```diff
@@ -633,11 +633,13 @@ def pull(self, options):
         Options:
             --ignore-pull-failures  Pull what it can and ignores images with pull failures.
             --parallel              Pull multiple images in parallel.
+            --quiet                 Pull without printing progress information
```
Comment (shin-) [inline compose/cli/main.py]: I think `--no-progress` might be a better, more descriptive name for this option. `--quiet` could mean a number of different things.

File: tests/acceptance/cli_test.py [hunk 2]
```diff
@@ -436,6 +436,9 @@ def test_pull_with_ignore_pull_failures(self):
         assert ('repository nonexisting-image not found' in result.stderr or
                 'image library/nonexisting-image:latest not found' in result.stderr)
 
+    def test_pull_with_quiet(self):
+        assert self.dispatch(['pull', '--quiet']).stdout == ''
```
Comment (shin-) [inline tests/acceptance/cli_test.py]: I don't think that test is correct - all Compose status messages are written on `stderr`, including progress bars.

**Comment by shin-** [comment]
Thank you for submitting this! 

It looks pretty good, but I have a few comments that need to be addressed before we merge this.

**Comment by jbarciauskas** [comment]
Thanks, I will get to them shortly. I wasn't sure about the test, I just
copied another existing one. I should have gone back to double check, sorry!

File: compose/cli/main.py (hunk 1 above)
Comment (jbarciauskas) [inline compose/cli/main.py]: Thinking about this again, it does suppress *all* output, and it is essentially a pass-through for a flag named "silent", so --quiet seems appropriate.

File: tests/acceptance/cli_test.py (hunk 2 above)
Comment (jbarciauskas) [inline tests/acceptance/cli_test.py]: The effect of setting silent=True is that all output is suppressed, so I will test both.

File: compose/cli/main.py (hunk 1 above)
Comment (jbarciauskas) [inline compose/cli/main.py]: @shin- had forgotten this was still lying around, happy to switch this if it's necessary to merge it

**Comment by shin-** [comment]
Thank you for bumping this. Do you mind squashing those commits together? Code looks good otherwise.

**Comment by jbarciauskas** [comment]
@shin- all set!

**Comment by shin-** [comment]
LGTM, thanks!

**Pull request merged**